### DISCUSSION TITLE
WebSocket反向连接403错误

### DISCUSSION THREAD

**Discussion by FairyReimu** [nonebot/discussions#201] [问答]
使用go-cqhttp连接时
nonebot版本为2.0.0-beta.5
错误信息如下：
uvicorn | ('127.0.0.1', 40470) - "WebSocket /onebot/v11/ws/" 403
[INFO] websockets | connection failed (403 Forbidden)
[INFO] websockets | connection closed

![image](https://user-images.githubusercontent.com/90098435/187154457-fe8c586e-fe82-4642-a8f6-1f8f2505aeb2.png)

**Comment by yanyongyu** [comment]
没有加载适配器吧，检查一下你的bot.py

**Reply by FairyReimu** [reply]
导入协议适配器，如OneBot V11，那from后面的名字需要填写成什么

**Reply by FairyReimu** [reply]
找到了，是nonebot.adapters.onebot.v11，感谢，已解决

**Reply by 1103859909** [reply]
大哥 我填了nonebot.adapters.onebot.v11，结果还是不行（愁
![image](https://user-images.githubusercontent.com/107859647/196025348-7e1a2af2-8f4c-4917-bed7-ebfbf0a1686b.png)
![image](https://user-images.githubusercontent.com/107859647/196025351-4d809408-3435-49cd-a8be-8f6f783baa47.png)

**Reply by yanyongyu** [reply]
`driver.register_adapter(Adapter)`

**Reply by 1103859909** [reply]
![image](https://user-images.githubusercontent.com/107859647/196692666-cf62964b-e1dc-46b0-b48f-5ab8540bf330.png)
QwQ

**Reply by lingyu667** [reply]
佬咋解决的

**Reply by lingyu667** [reply]
我还是报错

**Reply by lingyu667** [reply]
> ![image](https://user-images.githubusercontent.com/107859647/196692666-cf62964b-e1dc-46b0-b48f-5ab8540bf330.png)QwQ

解决了吗？

**Comment by lingyu667** [comment]
![Uploading 6af73d0c588aa8ec.jpg…]()

**Comment by 1103859909** [comment]
在用pip安装适配器



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2022年11月7日(周一) 中午11:32
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: WebSocket反向连接403错误




  
QwQ
  
解决了吗？
